blocks/results-list-block/features/results-list/results/index.jsx

```diff
@@ -102,22 +105,18 @@ const Results = ({
         }
         promo_items {
           basic {
+			_id
+            auth
```

Comment:
Should take a note from Matt on his update.  I think some others missed this as well:
![image](https://user-images.githubusercontent.com/2287238/200386758-882ecde3-baae-4e59-b915-64a23a044486.png)

Reply:
@nschubach Fixed it in [e9b1204](https://github.com/WPMedia/arc-themes-blocks/pull/1537/commits/e9b1204703519125aa93df005940c0c94cdbe4ba)